docarray/utils/lookup.py

```diff
@@ -0,0 +1,326 @@
+"""
```

Comment:
can we create a folder filter inside utils where we put the 3 files filter lookup and query parser ?

Reply:
I will